### PR TITLE
add note about SELinux and Docker Desktop

### PR DESCRIPTION
**Pull request by szaimen**
For https://github.com/nextcloud/all-in-one/discussions/485#discussioncomment-6088261
and https://github.com/nextcloud/all-in-one/issues/2450

### CODE REVIEW

File: Containers/mastercontainer/start.sh [hunk 1]
```diff
@@ -61,6 +61,8 @@ fi
 # Check if api version is supported
 if ! sudo -u www-data docker info &>/dev/null; then
     print_red "Cannot connect to the docker socket. Cannot proceed."
+    echo "If you are on Docker Desktop v4.19 or higher, see https://github.com/nextcloud/all-in-one/issues/2450"
+    echo "If SELinux is enabled on your host, see https://github.com/nextcloud/all-in-one#how-to-change-the-default-location-of-nextclouds-datadir#are-there-known-problems-when-selinux-is-enabled"
```
Comment (Zoey2936) [inline Containers/mastercontainer/start.sh:65]:
```suggestion
    echo "If SELinux is enabled on your host, see https://github.com/nextcloud/all-in-one#are-there-known-problems-when-selinux-is-enabled"
```

**Comment by Zoey2936** [comment]
maybe also add this option commented to the compose file

**Comment by Zoey2936** [comment]
also not sure if privileged or security opt is better

**Comment by szaimen** [comment]
> also not sure if privileged or security opt is better

I think privileged is the last restore so I would say security opt is more secure.

File: Containers/mastercontainer/start.sh (hunk 1 above)
Comment (szaimen) [reply Containers/mastercontainer/start.sh]: thanks, fixed!

**Comment by szaimen** [comment]
> maybe also add this option commented to the compose file

I would leave it out for now. If the question comes up more often after adding this doc, we can still adjust the compose.yaml later on.

**Comment by szaimen** [comment]
> > also not sure if privileged or security opt is better
> 
> I think privileged is the last resort so I would say security opt is more secure.

image

From https://danwalsh.livejournal.com/74095.html

**Comment by Zoey2936** [comment]
ahh I see

**Review by Zoey2936 — APPROVED** [review]

**Pull request merged**